src/utils/html.js

```diff
@@ -388,13 +389,8 @@ u.addMentionsMarkup = function (text, references, chatbox) {
     return text;
 };
 
-
-u.convertToHyperlink = function (url) {
-    const uri = getURI(url);
-    if (uri === null) {
-        return url;
-    }
```

Comment:
It's not clear to me why you removed these lines, can you please explain why?

Reply:
This `uri === null` check is now part of line `420`.